formatted_error.go

```
@@ -65,3 +69,12 @@ func indentedReason(reason string, level int) string {
 	}
 	return indented
 }
+
+func (f FastlaneRunner) formattedCommandErrorMessage(cmd command.Command, err error) string {
```

This function should return an error and let the caller functions handle the error (fmt.Sprintf -> fmt.Errorf).

Also, I suggest naming this function as `wrapCommandError` because here we create a new error with additional context.